src/controllers/recoveryController.ts

```
@@ -29,13 +49,27 @@ export async function postUnlock(
 ): Promise<void> {
   try {
     const body = unlockAppSchema.parse(req.body);
+    // Extract device fingerprint from request headers and body
+    const deviceFingerprint: DeviceFingerprint = {
+      userAgent: req.headers['user-agent'] as string || 'unknown',
```

ESLint

[error] Replace `req.headers['user-agent']·as·string·||·'unknown'` with `(req.headers["user-agent"]·as·string)·||·"unknown"`

(prettier/prettier)